### PR TITLE
Creates new migrations for providers and service entries

### PR DESCRIPTION
**Pull request by jaypdl**
Description
New migrations for the new DB schema

What work was done?
• created new migration files for providers table and service entries table
Why was this work done?
• In order to restructure the database to fit our desired schema
Type of change
• New feature (non-breaking change which adds functionality)
Change Status
• Completed, ready to review and merge
Checklist
• My code follows the style guidelines of this project
• I have performed a self-review of my own code
• My changes generate no new warnings
• There are no merge conflicts

### CODE REVIEW

File: data/newMigrations/017-service_entries.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    tbl
+      .uuid('service_entry_id')
+      .primary()
+      .defaultTo(knex.raw('gen_random_uuid ()'));
+    tbl
+      .integer('service_type_id')
+      .unsigned()
+      .notNullable()
+      .references('service_type_id')
+      .inTable('service_types');
```
Comment (samplesn08) [inline data/newMigrations/017-service_entries.js:12]: Don't forget the onUpdate and onDelete!

Comment (jaypdl) [reply data/newMigrations/017-service_entries.js:12]: By default onUpdate and onDelete are set to restrict, so it's only actually needed when you want to change to cascade.

Do you still want it added in?

Comment (samplesn08) [reply data/newMigrations/017-service_entries.js:12]: Oh!  My bad, that works

**Review by samplesn08 — APPROVED** [review]
Code is clean and readable, approved for merge!

**Pull request merged**